content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -397,7 +397,7 @@ ensure your kubelet services are started with the following flags:
 
 ## Device plugin integration with the Topology Manager
 
-{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+{{< feature-state for_k8s_version="v1.27" state="stable" >}}
```

Comment:
@ffroman,  as the TopologyManager [feature](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features) gate was stable in v1.27, so we have to do this change on a master branch so it will reflect on the latest k8s website (which is for k8s v1.27).

Reply:
I believe this needs a review from sig/node.

Reply:
I'd also prefer to see this line change for _main_

Reply:
I'm fine expunging this fix from this PR and file a new trivial one against `main`. I'll do in the coming days.
Thanks for your reviews and feedback.

Reply:
here: https://github.com/kubernetes/website/pull/41846